client/src/api/apiKeys.ts

```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
+
+export const getKeys = () => {
```

Comment:
need the serverUrl param. in order to separate the backend and the frontend to different origins.